layers/+lang/ruby/packages.el

```diff
@@ -116,6 +117,23 @@
       (evil-leader/set-key-for-mode 'enh-ruby-mode "msR" 'ruby-send-region-and-go)
       (evil-leader/set-key-for-mode 'enh-ruby-mode "mss" 'ruby-switch-to-inf))))
 
+(defun rspec/init-rspec-mode ()
+  (use-package rspec-mode
+    :defer t
+    :config
+    (progn
+      (spacemacs|diminish rspec-mode " Ƨ" " RSp")
+
+      (evil-leader/set-key-for-mode 'enh-ruby-mode
+        "mps" 'rspec-verify-single
+        "mpa" 'rspec-verify-all
+        "mpr" 'rspec-rerun
+        "mpb" 'rspec-verify-matching
+        "mpc" 'rspec-verify-continue
+        "mpm" 'rspec-verify-method
+        "mpt" 'rspec-toggle-example-pendingness
+        ))))
+
```

Comment:
Why the `mp` prefix?  Shouldn't this be under the `mt` prefix per the [conventions](https://github.com/syl20bnr/spacemacs/blob/develop/doc/CONVENTIONS.org#tests)